communications/telegram/telegram.go

```diff
@@ -141,14 +148,18 @@ func (t *Telegram) InitialConnect() error {
 		return errors.New(resp.Description)
 	}
 
-	warmWelcomeList := make(map[string]int64)
 	for i := range resp.Result {
-		if resp.Result[i].Message.From.ID != 0 {
-			warmWelcomeList[resp.Result[i].Message.From.UserName] = resp.Result[i].Message.From.ID
+		if resp.Result[i].Message.From.UserName != "" && resp.Result[i].Message.From.ID != 0 {
+			_, ok := t.AuthorisedClients[resp.Result[i].Message.From.UserName]
+			if !ok {
```

Comment:
I think this could be on `verbose` unless its a bit more important than I realise to highlight? eg me spamming 64 messages in a chatroom while unauthorised prints this error 64 times

Reply:
Security related messages should be relayed by default without verbosity enabled, however if a user spams your bot multiple times, then that shouldn't spam the logs. I've opted to instead report all unique unauth user warnings and then it's up to the user to ban repeat offenders